<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameForm</class>
 <widget class="QMainWindow" name="GameForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="PlayerMap2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QLabel" name="Map2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>971</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="LoginForm.qrc">:/newPrefix/2PlayerMap.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="Regions2" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>1001</width>
       <height>711</height>
      </rect>
     </property>
     <widget class="QLabel" name="Region20">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>320</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region24">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region25">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>430</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region26">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>530</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region27">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>570</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region28">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>470</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region29">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>480</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region210">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region211">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region21">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region22">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region23">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="PlayerMap3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QLabel" name="Map3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>971</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="LoginForm.qrc">:/newPrefix/3PlayerMap.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="Regions3" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>1001</width>
       <height>711</height>
      </rect>
     </property>
     <widget class="QLabel" name="Region30">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>320</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region34">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region35">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>430</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region36">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>530</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region37">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>570</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region38">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>520</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region39">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>480</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region310">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region311">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region31">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region32">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region33">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region312">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>420</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region313">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>220</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region314">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>130</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="PlayerMap4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QLabel" name="Map4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>971</width>
       <height>701</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="LoginForm.qrc">:/newPrefix/4PlayerMap.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="Regions4" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>1001</width>
       <height>711</height>
      </rect>
     </property>
     <widget class="QLabel" name="Region40">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>370</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region44">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>300</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region45">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>430</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region46">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>530</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region47">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>570</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region48">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>520</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region49">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>440</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region410">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region411">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>200</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region41">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>40</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region42">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region43">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region412">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>420</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region413">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>220</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region414">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>130</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region415">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>550</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region416">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>308</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Region417">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>81</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LoginForm.qrc">:/newPrefix/Castle icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="LoginForm.qrc"/>
 </resources>
 <connections/>
</ui>
